scipy/spatial/_spherical_voronoi.py

```diff
@@ -53,17 +41,20 @@ class SphericalVoronoi:
 
     Attributes
     ----------
-    points : double array of shape (npoints, 3)
-        the points in 3D to generate the Voronoi diagram from
+    points : double array of shape (npoints, ndim)
+        the points in `ndim` dimensions to generate the Voronoi diagram from
     radius : double
         radius of the sphere
-    center : double array of shape (3,)
+    center : double array of shape (ndim,)
         center of the sphere
-    vertices : double array of shape (nvertices, 3)
+    vertices : double array of shape (nvertices, ndim)
         Voronoi vertices corresponding to points
     regions : list of list of integers of shape (npoints, _ )
         the n-th entry is a list consisting of the indices
         of the vertices belonging to the n-th point in points
+    circumradii : double array of shape (nvertices,)
+        The radius of the spherical caps which cover each facet.  This is also
+        the distance from a Voronoi vertex to its nearest input points.
```

Comment:
This sounds a lot like the relationship between Delaunay and Voronoi. I find the usage of the circumscribed circle here a little confusing relative to the case in the Delaunay modification to produce its dual (Voronoi).

Reply:
I have removed this for now.